server/channels/app/user.go

```diff
@@ -624,6 +684,8 @@ func (a *App) GetUsersInChannelPageByAdmin(options *model.UserGetOptions, asAdmi
 	return a.sanitizeProfiles(users, asAdmin), nil
 }
 
+// GetUsersNotInChannel gets a page of users that are not in the specified channel and team.
```

Comment:
I think by filter it's referring to `viewRestrictions` otherwise it'd be wrong because teamId and channelId are required.